tests/test_io.py

```diff
@@ -428,7 +428,7 @@ def test_gwy_read_component(load_scan_dummy: LoadScans) -> None:
 @pytest.mark.parametrize(
     "load_scan_object, length, image_shape, image_sum, filename, pixel_to_nm_scaling",
     [
-        ("load_scan", 1, (1024, 1024), 30695369.188316286, "minicircle", 0.4940029296875),
+        ("load_scan_spm", 1, (1024, 1024), 30695369.188316286, "minicircle", 0.4940029296875),
```

Comment:
Could we add in `load_scan_data` as another set of parameters for this test to check the `.topostats`?

Reply:
👍 
I've added `load_scan_gwy` and `load_scan_topostats` (I forgot to add `load_scan_gwy` from my gwy file format PR 😨 )

Reply:
Good catch on the `load_scan_gwy`, wasn't even thinking about that.